Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+# @app.route('/add/<int:num>/<int:num2>')
+# def add(num, num2):
+# 	return str(num + num2)
+
+# @app.route('/subtract/<int:num>/<int:num2>')
+# def subtract(num, num2):
+# 	return str(num - num2)
+
+# @app.route('/divide/<int:num>/<int:num2>')
+# def divide(num, num2):
+# 	return str(num / num2)
+
+# @app.route('/multiply/<int:num>/<int:num2>')
+# def multiply(num, num2):
+# 	return str(num * num2)
+
+@app.route('/math/<operation>/<int:num>/<int:num2>')
+def math(operation, num, num2):
+	if operation == 'add':
```

Comment:
This is good; However, another way to do this is using a dictionary, just like you can in JavaScript using objects.

```python
def add(a, b):
    return a + b
def subtract(a, b):
    return a - b
def multiply(a, b):
    return a * b
def divide(a / b):
    return a / b

operations = {
    "add": add,
    "subtract": subtract,
    "multiply": multiply,
    "divide": divide
}